lib/galaxy/datatypes/converters/parquet_to_csv_converter.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+"""
+Input: parquet
+Output: csv
+"""
+import os
+import sys
+try:
+    from pyarrow import csv
+    from pyarrow import parquet
+except ImportError:
+    csv = parquet = None
+
+
+def __main__():
+    infile = sys.argv[1]
+    outfile = sys.argv[2]
+
+    if not os.path.isfile(infile):
+        sys.stderr.write(f"Input file {infile!r} not found\n")
+        sys.exit(1)
+
+    if csv is None or parquet is None:
```

Comment:
This is very minor. "csv" is a module in the standard library, so to interpret this line correctly one has to look for the import statement. Whereas if we just import pyarrow, not only `csv` would be unambiguous, we'd also have a very straightforward if-statement too:
```py 
if not pyarrow: 
    raise Exception("...pyarrow is not installed.")
```
Would you be terribly opposed to such a change?

Reply:
Something like this https://github.com/galaxyproject/galaxy/pull/13078/commits/1b8e5af7421b20c19b9fdc1fdccdc854155e7dac ?

Reply:
Yes, exactly! Thanks.